src/api.py

```diff
@@ -1422,11 +1423,22 @@ def HandleAdd(self, a, b):
         """Test two numeric params"""
         return a + b
 
+    @command('clearUISignalQueue')
+    def HandleclearUISignalQueue(self):
+        """clear UISignalQueue"""
+        queues.UISignalQueue.queue.clear()
+
     @command('statusBar')
     def HandleStatusBar(self, message):
         """Update GUI statusbar message"""
         queues.UISignalQueue.put(('updateStatusBar', message))
 
+    @command('getStatusBar')
+    def HandleGetStatusBar(self):
+        """Get GUI statusbar message"""
+        command, data = queues.UISignalQueue.get(block=False)
```

Comment:
It's a strange decision to add useless API commands only for testing purpose. Also the `UISignalQueue` receives various messages not related to the status bar.

Reply:
Ideally this command would only be available in test mode, as well as the flush one. But better to have some testing. There were several bugs related to queues in the past.